pennylane/qchem/observable_hf.py

```diff
@@ -143,20 +144,24 @@ def jordan_wigner(op):
     ([(0.5+0j), (-0.5+0j)], [Identity(wires=[0]), PauliZ(wires=[0])]) # corresponds to :math:`\frac{1}{2}(I_0 - Z_0)`
     """
     if len(op) == 1:
-        op = [((op[0], 1),)]
+        op = [(op[0], 1)]
 
     if len(op) == 2:
-        op = [((op[0], 1), (op[1], 0))]
+        op = [(op[0], 1), (op[1], 0)]
 
     if len(op) == 4:
-        op = [((op[0], 1), (op[1], 1), (op[2], 0), (op[3], 0))]
+        if notation is None:
```

Comment:
Should this be `physicist` instead of `None`?

Reply:
`physicist` does not make much sense for the one-body operators or single creation and annihilation operators. For the general case, `None` is used for now. The UI needs to be changed in the future to have the order as an argument, or use strings such as "4` 3` 2 1".

Reply:
I think since we don't support user input for a general order currently, it is better to use `physicist` instead of `None`. About the one-body terms, it just happens that they come out to be the same in both the notations, so they won't get affected (Paragraph above Eq 4. [here](http://vergil.chemistry.gatech.edu/notes/permsymm/permsymm.pdf)).

Reply:
A minor suggestion about the future UI - notation could take a third value called `arbitrary` along with the user can specify optional keyword argument `order` which would be a binary vector `[1001]`, where `1` is for the creation operator and `0` is for annihilation operator. This way for the other two standard notations, the user need not define the order.